### ISSUE TITLE
The detection effect of the model trained with the DUTS-TR data set is particularly poor

### ISSUE THREAD

**Issue by tongxiaozhong14** [milesial/Pytorch-UNet#223]
Dear author, thank you very much for the code you contributed. There is a big problem here.
In the training process, the training data set is DUTS-TR, the epoch is set to 1000, the learning rate is set to 1e-6, the scale is set to 1, and adam optimization is used. The trained model has the same effect of saliency detection. Not ideal, the author, is it because UNET network is not suitable for saliency detection? Or is there a problem with the parameter settings of my training?
I am very eager for you to answer my questions in time.
